secure_tunneling/include/aws/iotsecuretunneling/SecureTunnel.h

```diff
@@ -0,0 +1,100 @@
+#pragma once
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+#include <aws/crt/io/Bootstrap.h>
+#include <aws/crt/io/SocketOptions.h>
+#include <aws/iotdevice/secure_tunneling.h>
+#include <aws/iotsecuretunneling/Exports.h>
+
+int s_SecureTunnelingCallbackTest(Aws::Crt::Allocator *allocator, void *ctx);
+
+namespace Aws
+{
+    namespace Iotsecuretunneling
+    {
+        class AWS_IOTSECURETUNNELING_API DeviceApiHandle final
+        {
+          public:
+            DeviceApiHandle(Crt::Allocator *allocator) noexcept;
+            ~DeviceApiHandle();
+            DeviceApiHandle(const DeviceApiHandle &) = delete;
+            DeviceApiHandle(DeviceApiHandle &&) = delete;
+            DeviceApiHandle &operator=(const DeviceApiHandle &) = delete;
+            DeviceApiHandle &operator=(DeviceApiHandle &&) = delete;
+        };
+
+        // Client callback type definitions
+        using OnSecureTunnelingConnectionComplete = std::function<void(void)>;
```

Comment:
I don't think these names need to repeat the `SecureTunneling` part, they're already namedspaced.

Reply:
Done. Thanks for pointing this out.